pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/phimanifest/PhiManifestService.java

```diff
@@ -209,15 +211,23 @@ private boolean hasParticipantConsentedToSharedLearning(ElasticSearchParticipant
         }
         String dateOfBirth = participant.getDsm().get().getDateOfBirth();
         String dateOfMajority = (String) participant.getDsm().get().getDateOfMajority();
-        if (StringUtils.isBlank(dateOfMajority) || DateTimeUtil.isAdult(dateOfMajority)) {
-            return (Boolean)getAdultParticipantConsentedToTumorAnswer(participant, ddpInstanceDto.getStudyGuid())
+        String participantStatus = participant.getStatus().orElse("");
+        if (StringUtils.isBlank(dateOfMajority) || participant.hasCompletedActivity(CONSENT_ACTIVITY_CODE)) {
+            //self adult enrollment
+            return (Boolean) getAdultParticipantConsentedToTumorAnswer(participant, ddpInstanceDto.getStudyGuid())
                     .orElse(false);
+        } else {
+            //pediatric or aged-up in lost to followup
+            return isPediatricValidForPHI(participant, dateOfBirth, participantStatus);
         }
+    }
+
+    private static boolean isPediatricValidForPHI(ElasticSearchParticipantDto participant, String dateOfBirth, String participantStatus) {
         int age = DateTimeUtil.calculateAgeInYears(dateOfBirth);
         boolean hasCompletedPediatricConsentAddendum = participant.hasCompletedActivity(CONSENT_ADDENDUM_PEDIATRICS_ACTIVITY_CODE);
```

Comment:
Might be an edge case in existing code base ... if child was enrolled when < 7 and after few years current age will be > 7.. and the child might not have this question: SOMATIC_ASSENT_ADDENDUM_QUESTION_STABLE_ID answered ! (Need to test this scenario later & might need to fix it) .